src/api/lib/tasks/statistics/current.rake

```diff
@@ -0,0 +1,14 @@
+namespace :statistics do
+  desc 'Exports the current number of users, projects, packages, etc. to a file'
+  task current_numbers: :environment do
+    # The context of the file is NOT prepared to be used by R, just a easy way to get useful information
+    information = "OBS STATISTICS ON " + Date.today.to_s + "\n\n"
```

Comment:
This would be actually better: :+1: 

``` Ruby
    information = """OBS STATISTICS ON #{ Time.zone.today.to_s }\n
                  Number of users: #{ User.count.to_s }
                  Number of projects: #{ Project.count.to_s }
                  Number of packages: #{ Package.count.to_s }
                  Average time to accept a request: #{ time_in_hours.to_s } hours
                  """
```